code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
I mean i know you know yer code and all, but it will make this chem useless then! Right now we need a middle point between insta OP knock-out and other slowass knock-out. Makin this 10 just puts it in the other slow and useless chems list. Why even bother with this crap then when it only knocks-out on the 10th cycle, while capulettium puts to sleep on cycle 10, makes you look dead (scanners and pulse). While a sleep, you are blind, cannot hear, cannot talk and cannot move. Knock-out on cycle 10 or sleep at cycle 10? Pick is obvious. Stop hammering everything that seems OP to the ground FFS.